src/components/client/campaigns/CampaignInfo/CampaignInfoStatus.tsx

```diff
@@ -69,9 +69,7 @@ export default function CampaignInfoStatus({ campaign, showExpensesLink }: Props
       </Box>
       <InfoStatusWrapper>
         <Grid item xs={12} md={6}>
-          <StatusLabel>
-            {t(`campaigns:campaignTypesFields.${campaign.campaignType.slug}`)}
-          </StatusLabel>
+          <StatusLabel>{campaign.campaignType.name}</StatusLabel>
```

Comment:
We will loose the translation this way.
Can't we use campaignType.category?

Reply:
Done.